contrib/greetd/template.py

```diff
@@ -0,0 +1,53 @@
+pkgname = "greetd"
+pkgver = "0.9.0"
+pkgrel = 0
+build_style = "cargo"
+hostmakedepends = [
+    "bmake",
+    "cargo",
+    "pkgconf",
+    "scdoc",
+]
+makedepends = [
+    "linux-pam-devel",
+    "rust-std",
+]
+pkgdesc = "Minimal and flexible login manager daemon"
+maintainer = "natthias <natthias@proton.me>"
+license = "GPL-3.0-or-later"
+url = "https://git.sr.ht/~kennylevinsen/greetd"
+source = f"https://git.sr.ht/~kennylevinsen/greetd/archive/{pkgver}.tar.gz"
+sha256 = "a0cec141dea7fd7838b60a52237692d0fd5a0169cf748b8f8379d8409a3768eb"
+
+system_users = [
+    {
+        "name": "_greeter",
+        "id": None,
+        "groups": ["video"],
```

Comment:
is there a reason to do that? shouldn't a graphical greeter use elogind/libseat to request the device access? using the video group is a big hammer (it gives access to the gpu device node)

Reply:
Yea, it should, I just followed upstreams reccomendation, as well as what other distros do. Should I use elogind or libseat to manage this?

Reply:
greetd itself doesn't need it (it doesn't do anything with the gpu), this is only relevant to graphical greeters, and at least e.g. gtkgreet expects you to run it in a wayland compositor, which will use logind/seatd

Reply:
Yup, but with libseatd, the _greetd user would need the _seatd group, no?

Reply:
depends on the libseat backend; for logind you don't need anything (it uses dbus to communicate), for seatd you need to be in the group (it uses a unix domain socket to communicate, owned by `root:_seatd`)

Reply:
for now it's probably fine not to add it to any groups as elogind is the recommended stock solution, when it's time to kill elogind we'll need to figure something out